docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Understanding Lit PKPs as escrow
```

Comment:
> Understanding Lit PKPs as escrow

"Using Lit PKPs as escrow accounts"

Reply:
ig it should be the same on lines 4 and 8, should I make it escrow accounts at both places or end it at escrow?

Reply:
i think escrow is fine